server/highlight.py

```diff
@@ -121,15 +126,15 @@ def diff_files(files_before, files_after, diff_type):
             before = files_before.get(filename, '')
             after = files_after.get(filename, '')
             if len(before) > DIFF_SIZE_LIMIT or len(after) > DIFF_SIZE_LIMIT:
-                files[filename] = File([], too_big=True)
+                files[filename] = File(filename, [], after, too_big=True)
             else:
                 lines = list(highlight_diff(filename, before, after, diff_type))
-                files[filename] = File(lines)
+                files[filename] = File(filename, lines, after)
     else:
         for filename, source in files_after.items():
             if len(source) > DIFF_SIZE_LIMIT:
-                files[filename] = File([], too_big=True)
+                files[filename] = File(filename, [], too_big=True)
```

Comment:
Yup - I forgot to push eecb67e0029cfd842d50a27f2b8e906792ff570f which added default argument so this a bit neater.